dataactvalidator/config/sqlrules/fabs44_detached_award_financial_assistance_3.sql

```diff
@@ -10,10 +10,11 @@ FROM detached_award_financial_assistance AS dafa
 WHERE submission_id = {0}
     AND COALESCE(legal_entity_zip5, '') <> ''
     AND COALESCE(legal_entity_congressional, '') <> ''
-    AND dafa.legal_entity_congressional
-        NOT IN (SELECT sc.congressional_district_no
-    	        FROM state_congressional AS sc,
-    	 	        zips as z
-    	        WHERE z.zip5 = dafa.legal_entity_zip5
-    	            AND z.state_abbreviation = sc.state_code
-    	            AND sc.census_year IS NULL OR sc.census_year >= 2000);
+    AND NOT EXISTS (
+        SELECT 1
+        FROM state_congressional AS sc,
+            zips as z
```

Comment:
I missed this last time, which was my bad. I think we should explicitly use `join` here since that's how all our other SQL is written and we want to stay standardized. We don't usually do implicit joins. Also, all operators should be capitalized per our SQL standards, so the `as` should be caps.